index.html

```diff
@@ -90,25 +69,89 @@
 
     <script>
 
-      document.querySelector('.ramda-repl-target').addEventListener('click', function(e) {
+      (function() {
 
-        ramdaRepl(e.target, {
 
-          apiUrl: 'https://www.googleapis.com/urlshortener/v1/url?key=AIzaSyDhbAvT5JqkxFPkoeezJp19-S_mAJudxyk',
-          returnUrl: 'http://ramdajs.com/repl/',
+        var ramdaUrl;
 
-          // If unset, initialValue will use the value or textcontent of the
-          // target element.
-          initialValue: URI.decode(window.location.hash).substring(2),
-
-          // Called with the pre-compiled content
-          onChange : function(code) {
-            window.location.hash = '?' + URI.encode(code);
+        var rx = /^\d+\.\d+(\.\d+)?$/;
+        var version = 'latest';
+        location.search.substr(1).split('&').forEach(function(pair) {
+          var ps = pair.split('=');
+          if (ps[0] === 'v') {
+            if (rx.test(ps[1])) {
+              version = ps[1];
+            }
           }
+        });
+
+        var protocol = location.protocol === 'file:' ? 'https:' : location.protocol;
+
+        ramdaUrl = protocol + '//cdn.jsdelivr.net/ramda/' + version + '/ramda.min.js';
+
+        document.querySelector('.ramda-repl-target').addEventListener('click', function(e) {
+
+          ramdaRepl(e.target, {
+
+            apiUrl: 'https://www.googleapis.com/urlshortener/v1/url?key=AIzaSyDhbAvT5JqkxFPkoeezJp19-S_mAJudxyk',
+            returnUrl: 'http://ramdajs.com/repl/',
+
+            // If unset, initialValue will use the value or textcontent of the
+            // target element.
+            initialValue: URI.decode(window.location.hash).substring(2),
+
+            // Called with the pre-compiled content
+            onChange : function(code) {
+              window.location.hash = '?' + URI.encode(code);
+            },
+
+            // The following config defines scripts that will be dynamically
+            // loaded upon creating the REPL. Each has the following properties:
+            //
+            // src      - The source of the script, a URL
+            // global   - (Optional) a name of a global that the script introduces
+            // exposeAs - (Optional) a name that can alias the global
+            // expose   - (Optional) a list of methods to expose globally
+
+            // Required.
+            // As ramda may be a dependency for other scripts, it will
+            // be intentionally loaded before any other script.
+            ramdaScript: {
+              src    : ramdaUrl,
+              global : 'R'
+            },
+
+            // Optional.
+            // Here we can declare a list of libraries that we wish to have
+            // loaded and exposed in the repl.
+            scripts: [
+              {
+                src      : 'http://wzrd.in/standalone/sanctuary@latestx',
```

Comment:
Yes, that was careless of me. I added it to force an error so I could take a screenshot. I've removed the `x`.